payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/autocomplete/PlacesClientProxy.kt

```diff
@@ -0,0 +1,182 @@
+package com.stripe.android.ui.core.elements.autocomplete
+
+import android.content.Context
+import android.graphics.Typeface
+import android.text.style.StyleSpan
+import com.google.android.libraries.places.api.Places
+import com.google.android.libraries.places.api.model.AutocompleteSessionToken
+import com.google.android.libraries.places.api.model.TypeFilter
+import com.google.android.libraries.places.api.net.FetchPlaceRequest
+import com.google.android.libraries.places.api.net.FindAutocompletePredictionsRequest
+import com.google.android.libraries.places.api.net.PlacesClient
+import com.stripe.android.BuildConfig
+import com.stripe.android.ui.core.R
+import com.stripe.android.ui.core.elements.autocomplete.model.AddressComponent
+import com.stripe.android.ui.core.elements.autocomplete.model.AutocompletePrediction
+import com.stripe.android.ui.core.elements.autocomplete.model.FetchPlaceResponse
+import com.stripe.android.ui.core.elements.autocomplete.model.FindAutocompletePredictionsResponse
+import com.stripe.android.ui.core.elements.autocomplete.model.Place
+import kotlinx.coroutines.tasks.await
+
+internal interface PlacesClientProxy {
+    suspend fun findAutocompletePredictions(
+        query: String?,
+        country: String,
```

Comment:
Is this required btw?

Just thinking out loud we might not always have a country code to pass.

Reply:
When would we not have a country? If this is the case, we can remove it later!